content/_index.md

```diff
@@ -1,8 +1,80 @@
 +++
-title = "Snowplow Accelerator"
-menuTitle="Introduction"
+title = "Introduction"
+menuTitle = "Introduction"
+pre = "<i class='fas fa-rocket'></i> "
 chapter = false
 weight = 1
 +++
 
-This is an example of an Accelerator Introduction
\ No newline at end of file
+!['logo-banner'](images/logo_banner_2.png)
+
+Welcome to the **Fractribution** accelerator. This is an accelerator to demonstrate and guide you on how to run fractional attribution analysis on Snowplow data, coined 'Fractribution'. Fractional attribution allows you to attribute the value of a conversion to one or more channels depending on the conversion pathway. As a result, it becomes possible to determine the revenue per channel, as well as the return on advertising spend (ROAS) if you have spend data for each marketing channel. Once finished, you will be able to build a deeper understanding of customer behaviour on your website and use your data to influence business decisions.
```

Comment:
> Once finished, you will be able to build a deeper understanding of customer behaviour on your website and use your data to influence business decisions.

The above would sound a bit generic to me and not showing the actual end goal. I would go with something along the lines of:
`Once finished, you will be able to enhance your strategic marketing decisions, such as channel investment and focus, in a more data informed way.`

> if you have spend data
I would change it to something like: `once you have cost data` since it might resonate better with the audience.

> as well as the return on advertising spend (ROAS)
It is expected that people into performance marketing will be able to understand this term but I would suggest we add somewhere with an asterisk(*) the plain definition of ROAS. E.g.
`the amount of revenue that is earned for every dollar spent on a campaign`